src/renderer/Player.tsx

```diff
@@ -63,7 +63,8 @@ function Player(props: { mics: InputDevice[] }) {
                 break;
               case "YoutubeQueueItem":
                 setShouldShowPianoRoll(false);
-                videoRef.current.src = `${window.origin}/vids/${popSong.id}.mp4`;
+                const staticUrl = `http://localhost:8080/static`;
```

Comment:
Initially I had this as
`const staticUrl = `${window.location.protocol}//${window.location.hostname}:8080/static`;`
Which worked fine when running from `yarn start` but doesn't work in the portable exe for some reason
Hardcoding this to `http://localhost:8080/static` makes it work, but I don't know if this will break for funky local networks